modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -176,127 +169,113 @@ $ ls $HOME/clusterconfig/
-+
-For PXE or ISO boots, you can create the Ignition config
-and APPEND the `ignition.config.url=` option to identify the location of
-the Ignition config. You also need to append `ignition.firstboot ignition.platform.id=metal`
-or the `ignition.config.url` option will be ignored.
+When doing an {op-system} bare metal installation, there are two types of Ignition configs that you can provide, with different reasons for providing each one:
 
 * **Permanent install Ignition config**: Every bare metal {op-system-first} installation
 needs to pass one of the Ignition config files generated by `openshift-installer`,
-including `bootstrap.ign`, `master.ign` and `worker.ign`, to the carry out the
+such as `bootstrap.ign`, `master.ign` and `worker.ign`, to carry out the
 installation. It is not recommended to modify these files.
 +
-For PXE installs, you pass the Ignition configs on the APPEND line using the
+For PXE installs, you pass the Ignition configs on the `APPEND` line using the
 `coreos.inst.ignition_url=` option. For ISO installs, after the ISO boots to
 the shell prompt, you identify the Ignition config on the `coreos-installer`
 command line with the `--ignition-url=` option.
 +
-Instead of passing the location of an Ignition config via a kernel or
+Instead of passing the location of an Ignition config using a kernel or
 command-line option, you can embed an Ignition config into the ISO
 installer image. This allows you to do bare metal installs with the ISO,
 without requiring access to an HTTP server. See “Embedding an Ignition
 config in the {op-system-first} ISO” for details.
 
+* **Live install Ignition config**: This type must be created manually and should be avoided if possible, as it is not supported by Red Hat. With this method, the Ignition config passes to the live install medium, runs immediately upon booting, and performs setup tasks before and/or after the {op-system} system installs to disk. This method should only be used for performing tasks that must be performed once and not applied again later, such as with advanced partitioning that cannot be done using a MachineConfig.
++
+For PXE or ISO boots, you can create the Ignition config
+and `APPEND` the `ignition.config.url=` option to identify the location of
+the Ignition config. You also need to append `ignition.firstboot ignition.platform.id=metal`
+or the `ignition.config.url` option will be ignored.
+
 [id="installation-user-infra-machines-advanced_embedignition_{context}"]
 === Embedding an Ignition config in the {op-system-first} ISO
-The following procedures describe how to embed an Ignition config into
-the ISO so it is applied when the new installation first boots from disk.
+To embed an Ignition config named `worker.ign` into an ISO image, for example `rhcos-<version>-live.x86_64.iso`, you must:
+
+. Copy the image from the link:https://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/latest/[coreos-installer image mirror] page to a local directory.
```

Comment:
This is the latest and greatest, but I don't know whether what follows is accurate. @bgilbert @miabbott

Reply:
Thanks, @miabbott - I kept the steps fairly simple: download and extract.